apps/nextjs-website/src/lib/strapi/homepage.ts

```diff
@@ -109,6 +111,94 @@ const makeStrapiHomepagePopulate = () =>
       ecosystem: {
         populate: ['products.logo', 'solutions.icon', 'solutionsCta.link'],
       },
+      seo: {
+        fields: [
+          'id',
+          'metaTitle',
+          'metaDescription',
+          'keywords',
+          'metaRobots',
+          'metaViewport',
+          'canonicalURL',
+        ],
+        populate: {
+          metaImage: {
+            fields: [
+              'id',
+              'name',
+              'alternativeText',
+              'caption',
+              'width',
+              'height',
+              'hash',
+              'ext',
+              'mime',
+              'size',
+              'url',
+              'previewUrl',
+              'provider',
+              'provider_metadata',
+            ],
+            populate: {
+              formats: {
+                populate: {
+                  thumbnail: {
+                    fields: [
+                      'name',
+                      'hash',
+                      'ext',
+                      'mime',
+                      'width',
+                      'height',
+                      'size',
+                      'path',
+                      'url',
```

Comment:
Do we need all this lines? Isn't it possible to simplify the populate with a '*'?

Reply:
It is because of this https://forum.strapi.io/t/how-do-i-use-the-seo-plugin/20638/3, but found a better way to have all the fileds, committing it now